### DISCUSSION TITLE
Alguma dica sobre somo adaptar o uso do novo `get_account_statements_paginated` ao que já estava sendo utilizado antes?

### DISCUSSION THREAD

**Discussion by felipythondev** [andreroggeri/pynubank#356] [Q&A]
Opa! Estou aqui adaptando [meu código](https://github.com/lipemorais/financas-automatizadas/blob/5bcce7f9ffe15a3352765301de6f91001bafc657/financas_automatizadas/main.py#L31) ao novo método de pegar as transações da conta. Como a resposta é muito diferente vou ter que mudar bastante coisas. Estava utilizando a própria estrutura de dados do Nubank mas com o novo método mudou muita coisa to achando meio inviável não ter que mudar tanto. 

Existe algum caminho mais simples do que só mudar tudo a partir da nova estrutura de retorno da API? Alguma função que traz da estrutura nova para a antiga ou algo do tipo?
Se esse realmente for o caminho, vou ver se consigo fazer um parser que funcionará feito um adapter(de ports e adapter)

**Comment by pecampelo** [comment]
Uma pergunta: isto alterou somente as transações de cartão de crédito ou as de contas corrente também, @lipemorais?

**Reply by felipythondev** [reply]
Essa foi uma alteração somente na conta. O cartão de crédito segue igual. [Aqui](https://github.com/andreroggeri/pynubank/blob/main/examples/extract-all-paginated-feed-pages.md) tem algumas explicações.

**Comment by leonardo-sinvus** [comment]
Fala pessoal, 
quando eu chamo feed = nu.get_account_feed_paginated(), estou tendo um erro  'The authentication method used doest not allow access to this resource', para as demais chamadas no cartão de crédito esta ok. 
Alguém teve esse mesmo erro ?

**Reply by leonardo-sinvus** [reply]
está sendo discutido nessa issue: #419
